terminal_reader.go

```diff
@@ -24,18 +37,18 @@ type TerminalReaderListParams struct {
 // TerminalReader is the resource representing a Stripe terminal reader.
 type TerminalReader struct {
 	APIResource
-	Deleted         bool              `json:"deleted"`
-	DeviceSwVersion string            `json:"device_sw_version"`
-	DeviceType      string            `json:"device_type"`
-	ID              string            `json:"id"`
-	IPAddress       string            `json:"ip_address"`
-	Label           string            `json:"label"`
-	Livemode        bool              `json:"livemode"`
-	Location        string            `json:"location"`
-	Metadata        map[string]string `json:"metadata"`
-	Object          string            `json:"object"`
-	SerialNumber    string            `json:"serial_number"`
-	Status          string            `json:"status"`
+	Deleted         bool                     `json:"deleted"`
+	DeviceSwVersion string                   `json:"device_sw_version"`
+	DeviceType      TerminalReaderDeviceType `json:"device_type"`
```

Comment:
that's a breaking change right?

Reply:
No, `string -> enum` in stripe-go is not breaking because `TerminalReaderDeviceType` is just a type alias for string (see line 9).

Here's a Go playground as demonstration: https://play.golang.org/p/5S-W7uC7pbD